src/main/java/cz/muni/fi/pa165/bluebat/entity/Trip.java

```diff
@@ -1,5 +1,92 @@
 package cz.muni.fi.pa165.bluebat.entity;
 
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.JoinTable;
+import javax.persistence.OneToMany;
+import javax.persistence.OneToOne;
+import javax.persistence.OrderBy;
+import java.time.LocalDate;
+import java.util.Collections;
+import java.util.List;
+import java.util.Objects;
+import java.util.Set;
+
+/**
+ * @author Ondrej Vaca
+ * Class representing a Trip entity
+ */
+
+@Entity
+@Setter
+@ToString
 public class Trip {
 
+    @Id
+    @GeneratedValue(strategy= GenerationType.IDENTITY)
+    @Getter
+    private Long id;
+
+    @Column(nullable = false)
+    @Getter
+    private LocalDate dateFrom;
+
+    @Column(nullable = false)
+    @Getter
+    private LocalDate dateTo;
+
+    @Column(nullable = false)
+    @Getter
+    private String destination;
+
+    @Column(nullable = false)
+    @Getter
+    private int availableTrips;
+
+    @OneToMany()
+    @OrderBy("validFrom DESC")
+    @JoinTable(name = "TRIP_PRICE")
+    private Set<Price> price;
```

Comment:
Promiň, ale špatně jsem se vyjádřil. Podle mého názoru by bylo lepší, kdyby to byl list a ne set, protože nám právě jde o to pořadí, které získáme z DB díky OrderBy. Set sám o sobě nezaručuje pořadí si myslím 🤔